### PR TITLE
Support sync for resources.

### PR DESCRIPTION
**Pull request by ahmetoz**

### CODE REVIEW

File: extension/test/unit/config/init/ensure-resources.spec.js [hunk 1]
```diff
@@ -61,27 +61,26 @@ describe('Ensure resources', () => {
     )
   })
 
-  it('should skip creating resources when they already exist', async () => {
```
Comment (lojzatran) [inline extension/test/unit/config/init/ensure-resources.spec.js:64]: Why did you remove this test?

**Review by lojzatran — APPROVED** [review]

File: extension/test/unit/config/init/ensure-resources.spec.js (hunk 1 above)
Comment (ahmetoz) [reply extension/test/unit/config/init/ensure-resources.spec.js:64]: it will not skip anymore instead it will check the diff and create actions, I tried to use but those tests are mostly based on mocking the API calls and error-prone, I wrote an integration test instead, unit tests might check error handling logic.
